src/Sulu/Bundle/WebsiteBundle/Tests/Application/Kernel.php

```diff
@@ -24,8 +24,6 @@ class Kernel extends SuluTestKernel
     public function __construct($environment, $debug, $suluContext = self::CONTEXT_ADMIN)
     {
         parent::__construct($environment, $debug, $suluContext);
-
-        $this->name = $suluContext;
```

Comment:
this should already be handled by the SuluKernel here:

https://github.com/sulu/sulu/blob/337c078aef79e860ef955f5994c5e1adb554aa6f/src/Sulu/Component/HttpKernel/SuluKernel.php#L65-L67